unlock-app/src/createUnlockStore.js

```diff
@@ -36,13 +32,12 @@ export default function createUnlockStore(config, browserHistory) {
   }, loadState())
 
   const middlewares = [
-    routerMiddleware(browserHistory),
-    lockMiddleware,
+    lockMiddleware
```

Comment:
Hey, we use dangling comas all over as part of our style guide, this avoid adding unneeded changes when a new items is added to an array/object.

Reply:
Will update it.